<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>update_gain</class>
 <widget class="QWidget" name="update_gain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>286</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>286</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>286</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>修改信息</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>修改小组获得的成果</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>小组编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>成果编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton
{
	background-color: #F8F8FF; /*背景色*/ 
	border-style: outset;
	border-width: 2px;
	border-radius:5px; /*边界圆滑*/
	border-color: #DCDCDC;
	font: bold 14px;
	min-width:2em;
	color:black; /*字体颜色*/
	font-family:宋体;
	padding: 5px;
}
 
#pushButton:hover
{
	background-color: rgb(0, 150, 0);
}
 
#pushButton:pressed
{
	background-color: #1E90FF; /*伪状态经过时背景色*/ 
	border-style: inset;
}
#pushButton:!enabled{
	background-color: rgb(100, 100, 100);
	border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2
{
	background-color: #F8F8FF; /*背景色*/ 
	border-style: outset;
	border-width: 2px;
	border-radius:5px; /*边界圆滑*/
	border-color: #DCDCDC;
	font: bold 14px;
	min-width:2em;
	color:black; /*字体颜色*/
	font-family:宋体;
	padding: 5px;
}
 
#pushButton_2:hover
{
	background-color: #ff0000;
}
 
#pushButton_2:pressed
{
	background-color: #1E90FF; /*伪状态经过时背景色*/ 
	border-style: inset;
}
#pushButton_2:!enabled{
	background-color: rgb(100, 100, 100);
	border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>小组名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>成果名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>待审批</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>通过</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>不通过</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,165,0);</string>
   </property>
   <property name="text">
    <string>审批意见：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_materiall">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#show_materiall
{
	background-color: #F0F8FF; /*背景色*/ 
	border-style: outset;
	border-width: 2px;
	border-radius:5px; /*边界圆滑*/
	border-color: #6495ED;
	font: bold 14px;
	min-width:2em;
	color:black; /*字体颜色*/
	font-family:楷体;
	padding: 5px;
}
 
#show_materiall:hover
{
	background-color: #00CED1;
}
 
#show_materiall:pressed
{
	background-color: #1E90FF; /*伪状态经过时背景色*/ 
	border-style: inset;
}
#show_materiall:!enabled{
	background-color: rgb(100, 100, 100);
	border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>查看相关证明材料</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
